src/app/right-hand-panel.js

```diff
@@ -13,20 +31,47 @@ import 'bootstrap';
 @customElement('right-hand-panel')
 @inject(Context, Element)
 export class RightHandPanel {
+
+    tabs = [
```

Comment:
I think what you are doing here is not a bad idea per se, I'm just thinking that at the moment to list and keep the tabs in the js (as well as the selected/default tab) is a bit too much. This method would make sense especially if it is used in the template to construct the tabs flexibly and lots of its properties are integrated into the corresponding html and these properties come from sources such as requests, i.e. based on data. As is we do reduce the template code with this loop which is nice I admit but we set up additional bindings with the repeat that have to be processed when the only thing that changes is the active class.

The other, bigger problem with this approach is that we still need a tabcontent for each tab item, so in a sense we achieve only half the work with the repeat, the rest is still static in the template and for it to work in any case one would have to adjust the code in the template regardless.

I know it doesn't look as nice in the template but I'd maybe go for an approach where I list the tabs explicitly and solely in html (without binding to the tab member in the js and repeat) but compare to the default/selected member in each tag to determine whether to put the active class there:
```
class="tab-pane ${selected_tab === 'settings' ? 'active' : ''}"
```

Has the nice side-effect that the loop you use is not necessary for active either and if in the class you set selected_tab to 'info' it should use that as a default.